addons/evm/src/commands/actions/call_contract.rs

```diff
@@ -48,91 +48,106 @@ lazy_static! {
                 documentation: "A description of the transaction.",
                 typing: Type::string(),
                 optional: true,
-                interpolable: true
+                tainting: false,
+                internal: false
             },
             rpc_api_url: {
-              documentation: "The URL of the EVM API used to broadcast the transaction.",
-              typing: Type::string(),
-              optional: true,
-              interpolable: true
+                documentation: "The URL of the EVM API used to broadcast the transaction.",
+                typing: Type::string(),
+                optional: true,
+                tainting: false,
+                internal: false
             },
             signer: {
                 documentation: "A reference to a signer construct, which will be used to sign the transaction.",
                 typing: Type::string(),
                 optional: false,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             contract_address: {
                 documentation: "The address of the contract being called.",
                 typing: Type::addon(EVM_ADDRESS),
                 optional: false,
-                interpolable: true
+                tainting: true,
+                internal: false
             },
             contract_abi: {
                 documentation: "The contract ABI, optionally used to check input arguments before sending the transaction to the chain.",
                 typing: Type::addon(EVM_ADDRESS),
                 optional: true,
-                interpolable: true
+                tainting: true,
```

Comment:
I'm not sure if this should be tainting - it is just used for type checking, to improve the error specificity. Chances are, if you change this, you'll be changing some _other_ field that _is_ tainting, but this one itself is not.

Reply:
```suggestion
                tainting: false,
```